.github/workflows/ci.yml

```diff
@@ -10,6 +10,15 @@ on:
     branches: [main, dev]
 
 jobs:
+  no-pdb:
```

Comment:
huh, I could've sworn I already added that

Reply:
Was a rebasing artifact, now this change is not in the PR.